### PR TITLE
Rs 2022.11.09

### PR DESCRIPTION
**Pull request by rschlaikjer**
2022-11-11-220942_1920x1140_scrot

### CODE REVIEW

File: script/Ciel/Day 9/09_01_CIEL09_6.txt [hunk 1]
```diff
@@ -861,44 +972,55 @@ But... something isn't right.
 [sha:1908a8ca7871caed8a04d06d17056fd40dd128c1]{
 -- Page 606, Offset 23689.
 -- 本来なら取るに足りない些細な間違いが、アイツを違う生き物に変えている―――
+// Unsure what exactly the 間違い is here. Biting her finger? Rising to Ciel's bait?
```
Comment (Petrikow) [inline script/Ciel/Day 9/09_01_CIEL09_6.txt:975]: ラブ
God this is such a big topic I don't even know where to begin.
So I won't.

File: script/Ciel/Day 9/09_01_CIEL09_6.txt [hunk 2]
```diff
@@ -861,44 +972,55 @@ But... something isn't right.
...
 Something that should've been a trifling blunder has instead transformed her into a different being.
 }
 [sha:bd88ed1203aeb5d5c4991af2029f19c486926747]{
 -- Page 607, Offset 23690.
 -- 「自由に、思うままに行動するというのですね。貴女が何の誓約もなく、思うままにロアを処罰すると。
 // random bullshit go (2)
-"To be free to act as you please, huh. If you were to punish Roa as you see fit, without forming any covenant, then―――
+// Mirroring Arcueid's 自由に from 23667.
+// Any lore behind 誓約 or is it just 'holding yourself to a standard'?
```
Comment (Petrikow) [inline script/Ciel/Day 9/09_01_CIEL09_6.txt:983]: She used to follow her 'instructions' even after the other TA were dead. Wake up, kill, etc.
Until now.

File: script/Ciel/Day 9/09_01_CIEL09_6.txt [hunk 3]
```diff
@@ -908,12 +1030,12 @@ Besides―――"
...
+"Can you think of anything more gratifying than tearing a woman you can't stand limb from limb?"
 }
 [sha:56b20363492cc66dafdadf30d5e578a77e92685e]{
 -- Page 611, Offset 23699.
 -- ―――始まりは一瞬だった。
-In the blink of an eye, the battle commences.
+The battle begins faster than I can even blink.
 }
 [sha:78ed2bd22c51dd5f80af79d8947686ec83241749]{
 -- Page 611, Offset 23700.
```
Comment (Petrikow) [inline script/Ciel/Day 9/09_01_CIEL09_6.txt:1041]: 白い化身 has long been avatar in white.

File: script/Ciel/Day 9/09_01_CIEL09_6.txt [hunk 4]
```diff
@@ -928,5 +1050,5 @@ As though anticipating her move, a hail of swords comes down raining on her.
 [sha:ad98ecc5a4356eeaf7c89a9b4b033772a197737a]{
 -- Page 611, Offset 23702.
 -- 俺が制止する隙間すらなく、二つの影は交戦を開始した。
-The two hazy silhouettes begin to clash without ever giving me a chance to intervene.
+Both figures leapt into the fray so quickly that I never even had a chance of stopping them.
```
Comment (Petrikow) [inline script/Ciel/Day 9/09_01_CIEL09_6.txt:1053]: Focus mismatch.

>Before I can so much as try to stop it, the battle between the two figures commence.

**Review by Petrikow — COMMENTED** [review]
Don't let the fact that pekora bought a fucking monkey distract you from the fact that this review has 69 fucking comments

File: script/Ciel/Day 9/09_01_CIEL09_6.txt [hunk 5]
```diff
@@ -694,124 +783,142 @@ Despite that, I can't let those words she just spoke slide.
...
 -- 「ええ、その通りよシエル！
-"Yes, that's right, Ciel!
+"Yes, that's the spirit, Ciel!
 }
 [sha:db0daea33860a407a7cc6d15eb3ccbc19833ba33]{
 -- Page 597, Offset 23665.
 -- これは吸血衝動とは違う。だって楽しいわ。すごく、すごく<心|むね>が弾んでいるの。まるで春の野原のよう！
-This is different from my vampiric urge. After all, I'm having fun! My heart is singing with such joy, it's almost as if I'm standing in the spring wilderness!
+// Man this まるで春の野原のよう is a weird one
+// I interpret it as her heart being fresh, green and flowering? The spring of new feelings?
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Ciel/Day 9/09_01_CIEL09_6.txt:840]: I guess do you think this is fine in that regard? Or something like 
> It's like my chest has become a blossoming wonderland!

or is the のよう associated more like
> I feel like I'm dashing through a blossoming meadow!

[the remark this answers is not in the document]
Comment (Petrikow) [reply script/Ciel/Day 9/09_01_CIEL09_6.txt:835]: Both are a bit long, but yes, the latter is more akin to what I was thinking.
Maybe not quite so long.

**Pull request merged**